tests/indexes/tests.py

```diff
@@ -25,12 +25,12 @@ def test_index_name_hash(self):
         """
         Index names should be deterministic.
         """
-        with connection.schema_editor() as editor:
```

Comment:
I don't follow _Why_ these changes here?

Reply:
@carltongibson they `__enter__` a `SchemaEditor` within a transaction (because of the `TestCase` transaction wrapping) which on SQLite as that would prevent proper constraint disabling. There's no need to `__enter__` these schema editor instances at all to call this method as the following `test_index_together` and `test_index_together_single_list` tests demonstrate.